Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift

```diff
@@ -1584,7 +1706,67 @@ class VisualRecognitionTests: XCTestCase {
                 XCTAssertEqual("Invalid image file", Error.localizedDescription)
                 expectation.fulfill()
             },
```

Comment:
Also, this is nitpicky, but perhaps we can pull the failure function out, like we do below. Keep things consistent :)